oshi-core/src/main/java/oshi/hardware/platform/windows/WindowsCentralProcessor.java

```diff
@@ -79,6 +79,28 @@
     private static final String PROCESSOR = "Processor";
 
     private Map<String, Integer> numaNodeProcToLogicalProcMap;
+    
+    private static final long BOOTTIME;
+    static {
+    	boolean found = false;
+        EventLogIterator iter = new EventLogIterator(null, "System", WinNT.EVENTLOG_BACKWARDS_READ);
+        while (iter.hasNext()) {
+            EventLogRecord record = iter.next();
+            if (record.getRecord().EventID.getLow().intValue() == 6005) {
+                if (found) {
+                    // Didn't find EventID 12, return first 6005
+                    break;
+                }
+                // First 6005; tentatively assign and look for EventID 12
+                BOOTTIME = record.getRecord().TimeGenerated.longValue();
+                found = true;
+            } else if (found && record.getRecord().EventID.getLow().intValue() == 12) {
+                // First 12 after 6005, this is boot time
+                BOOTTIME = record.getRecord().TimeGenerated.longValue();
+                break;
+            }
+        }
```

Comment:
You need to handle the case where the final variable isn't initialized in the loop.